### PR TITLE
2021-11-3-Progettare servizi pubblici basati su API

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Create: feat 2021-11-3-Progettare servizi pubblici basati su API.md
- type must be lower-case
- type must be one of [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test]
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2fb98e35cd2efa7e651f9037954ab7cc105a4841
</p>

File: _posts/it/2021-11-3-Progettare servizi pubblici basati su API [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Progettare servizi pubblici basati su API"
+subtitle: Il racconto della Community call del 27 ottobre e le esperienze dei partecipanti 	
+date: 2021-11-3 13:00:00 +0200
```
Comment (bfabio) [inline _posts/it/2021-11-3-Progettare servizi pubblici basati su API:5]:
```suggestion
date: 2021-11-03 13:00:00 +0200
```

File: _posts/it/2021-11-3-Progettare servizi pubblici basati su API [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+tags: developers designers API
```
Comment (bfabio) [inline _posts/it/2021-11-3-Progettare servizi pubblici basati su API:6]:
```suggestion
tags: developers designers api
```

File: _posts/it/2021-11-3-Progettare servizi pubblici basati su API [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+In conclusione, come di consueto, uno spazio per le domande dei partecipanti ricco di spunti e riflessioni. 
+
+Se vuoi approfondire, scarica la presentazione condivisa e [guarda il video](https://www.youtube.com/watch?v=aTBPk56SK3I){:target="_blank"} dell’incontro sul canale YouTube del Dipartimento per la trasformazione digitale.
+
+**Resta in contatto con la community*: raccontaci su [slack](https://app.slack.com/client/T6C27AXE0/CDKBYTG74){:target="_blank"} (#api), via mail (contatti@developers.italia.it) o su [Forum Italia](https://forum.italia.it/c/piano-triennale/interoperabilita/12){:target="_blank"} (Piano Triennale - Interoperabilità) altre API sviluppate secondo il modello di interoperabilità, per arricchire il catalogo di Developers Italia a beneficio di tutti! Oppure, segnalaci servizi basati su API che ritieni ad alto impatto per il numero di enti o casi d’uso che potrebbero beneficiarne.
+
+È già il momento di annunciare la prossima data: **le community call tornano il 24 novembre**. Ti aspettiamo!
+
```
Comment (bfabio) [inline _posts/it/2021-11-3-Progettare servizi pubblici basati su API:35]:
```suggestion
```

File: _posts/it/2021-11-3-Progettare servizi pubblici basati su API [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+È poi intervenuto Emiliano Verga di Cefriel, coordinatore scientifico dell’Ecosistema Digitale E015 della **Regione Lombardia**. Si tratta di un ambiente digitale, originariamente nato in occasione di Expo 2015, basato su un insieme di regole e processi che favoriscono l’incontro tra domanda e offerta di API tra soggetti pubblici e privati in diversi settori: turismo, mobilità sostenibile, ambiente, sicurezza e sanità.
+
+A seguire Walter Manni, Celestina Paglia (Dipartimento di Web Strategies, Research & Innovation dell’Azienda Promozione Turismo della Regione **Emilia-Romagna**) e Stefano Vena (Altrama s.rl) ci hanno parlato della realizzazione del servizio di condivisione di informazioni contenuto nel **portale turistico della Regione** tramite API. Uno strumento che ha consentito di semplificare la gestione e il trasferimento delle informazioni ai vari punti di pubblicazione locali.
```
Comment (bfabio) [inline _posts/it/2021-11-3-Progettare servizi pubblici basati su API:20]:
```suggestion
A seguire Walter Manni, Celestina Paglia (Dipartimento di Web Strategies, Research & Innovation dell’Azienda Promozione Turismo della Regione **Emilia-Romagna**) e Stefano Vena (Altrama s.r.l.) ci hanno parlato della realizzazione del servizio di condivisione di informazioni contenuto nel **portale turistico della Regione** tramite API. Uno strumento che ha consentito di semplificare la gestione e il trasferimento delle informazioni ai vari punti di pubblicazione locali.
```

**Review by bfabio — CHANGES_REQUESTED** [review]
We should rename the file to `2021-11-03-progettare-servizi-pubblici-basati-su-api.md`.

**Review by Francescopog — COMMENTED** [review]
changed title

**Review by bfabio — APPROVED** [review]

**Pull request merged**